### PR TITLE
Color Customization for Inline Result Bubbles

### PR DESCRIPTION
**Pull request by gobuki**
This will help with accessibility and reasoning about code structure.

To consider :
- Naming of configuration strings, which will be hard to deprecate.
- Whether to allow users to bind colors to [theme colors](https://code.visualstudio.com/api/references/theme-color). The `vscode.` prefix is necessary because programmatically verifying theme colors is unsupported. Prefixing seems mostly consistent with [other areas](https://code.visualstudio.com/api/extension-guides/webview#theming-webview-content) of the API.

Other notes : 
- Currently proposes that we refer to the left border as "accent", since at some point we may want to somehow support an all-around border -- these seem to be more central in high contrast color themes. 
- In settings, I wanted this to follow logically after `Julia > Execution : Result Type`, but couldn't find a way alphabetically.

Thanks to @pfitzseb for pointing me to the necessary resources.

Screen Shot 2021-01-06 at 11 01 51 AM

### CODE REVIEW

**Review by pfitzseb — COMMENTED** [review]
Thanks for the contribution! I left a few comments inline, but can also address them myself if you'd prefer this to just get merged.

File: src/interactive/results.ts [hunk 1]
```diff
@@ -86,25 +86,48 @@ export class Result {
 
     createResultDecoration(): vscode.DecorationRenderOptions {
 
-        const borderColor = this.content.isError ? '#d11111' : '#159eed'
+        const section = vscode.workspace.getConfiguration('julia')
+        const colorConfig = section.get<object>("execution.inlineResults.colors")
+
+        let colorFor = function (candidates: string[], defaultCallBack: () => string | vscode.ThemeColor): string | vscode.ThemeColor {
```
[the remark this answers is not in the document]
Comment (gobuki) [reply src/interactive/results.ts:92]: Ah, this was something I meant to mention in my request. Yes, it looks like a container for the string.

**Comment by gobuki** [comment]
Hey about to do a force push to fix those commit messages.

**Comment by gobuki** [comment]
I think that covers everything. Thanks for the reviews so far! 

I'm in a different timezone, so if I'm slow responding please commit and merge as you see fit.

**Review by pfitzseb — APPROVED** [review]

**Comment by pfitzseb** [comment]
Nice, great to have this in!

**Comment by gobuki** [comment]
Thanks!

**Pull request merged**